app/presenters/iiif_print/work_show_presenter_decorator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module IiifPrint
+  module WorkShowPresenterDecorator
+    # OVERRIDE: Hyrax 2.9.6
+    def total_pages(members = nil)
```

Comment:
Maybe we should consider bringing this over in a separate pr and research the purpose of it? and or bring over all the pieces needed for it, since I don't see total_pages anywhere else in this PR. 

ref PR for the other pieces needed to full implement it: https://github.com/scientist-softserv/louisville-hyku/commit/e25c01398cf47cf09ed24c6c16b6dba12f2ffd2f

Reply:
i think [this](https://github.com/samvera/hyrax/blob/main/app/presenters/hyrax/work_show_presenter.rb#L204-L206) is the method it's overriding

Reply:
I've removed it from this PR